docs/roman/references_general/flat_reffile.inc

```diff
@@ -4,7 +4,7 @@ FLAT Reference File
 -------------------
 
 :REFTYPE: FLAT
-:Data model: `~romancal.datamodels.FlatModel`
+:Data model: `roman_datamodels.FlatModel`
```

Comment:
This seems to be named [FlatRefModel](https://github.com/spacetelescope/roman_datamodels/blob/main/src/roman_datamodels/datamodels.py#L282)

Reply:
fixed